commit 5fb35a3a9bc482110b2c6afd1a5004909fa12501
Author: Weston Andros Adamson <dros@netapp.com>
Date:   Thu May 10 15:31:10 2012 -0400

    nfsd: return 0 on reads of fault injection files
    
    debugfs read operations were returning the contents of an uninitialized u64.
    
    Signed-off-by: Weston Andros Adamson <dros@netapp.com>
    Signed-off-by: J. Bruce Fields <bfields@redhat.com>

diff --git a/fs/nfsd/fault_inject.c b/fs/nfsd/fault_inject.c
index 9559ce468732..e6c38159622f 100644
--- a/fs/nfsd/fault_inject.c
+++ b/fs/nfsd/fault_inject.c
@@ -58,6 +58,7 @@ static int nfsd_inject_set(void *op_ptr, u64 val)
 
 static int nfsd_inject_get(void *data, u64 *val)
 {
+	*val = 0;
 	return 0;
 }